migration/iD11/oracle/202311071130_IDEMPIERE-5913.sql

```diff
@@ -0,0 +1,13 @@
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- Nov 7, 2023, 11:30:09 AM WIB
+INSERT INTO AD_Process_Para (AD_Process_Para_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,Name,AD_Process_ID,SeqNo,AD_Reference_ID,IsRange,FieldLength,IsMandatory,DefaultValue,ValueMin,ColumnName,IsCentrallyMaintained,EntityType,AD_Process_Para_UU,IsEncrypted,IsAutocomplete,DateRangeOption,IsShowNegateButton) VALUES (200447,0,0,'Y',TO_TIMESTAMP('2023-11-07 11:30:08','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2023-11-07 11:30:08','YYYY-MM-DD HH24:MI:SS'),100,'Keep Log Days',toRecordId('AD_Process','877068e7-ee18-4efb-b3ef-f6b341ce2189'),10,11,'N',0,'Y','7','0','KeepLogDays','N','D','05b81f27-19df-4758-9a49-ffaeab4d6287','N','N','D','N')
```

Comment:
Same here:
Instead of the toRecordId I think is better to insert directly the 200156